examples/tutorial/tutorial_notebooks/introduction/omnisci_on_native/exercise_1.ipynb

```diff
@@ -0,0 +1,227 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Modin is a drop-in replacement for pandas that distributes the computation \n",
+    "across all of the cores in your machine or in a cluster.\n",
+    "In practical terms, this means that you can continue using the same pandas scripts\n",
+    "as before and expect the behavior and results to be the same. The only thing that needs\n",
+    "to change is the import statement. Normally, you would change:\n",
+    "\n",
+    "```python\n",
+    "import pandas as pd\n",
+    "```\n",
+    "\n",
+    "to:\n",
+    "\n",
+    "```python\n",
+    "import modin.pandas as pd\n",
+    "```\n",
+    "\n",
+    "Changing this line of code will allow you to use all of the cores in your machine to do computation on your data. One of the major performance bottlenecks of pandas is that it only uses a single core for any given computation. Modin exposes an API that is identical to Pandas, allowing you to continue interacting with your data as you would with Pandas. There are no additional commands required to use Modin locally. Partitioning, scheduling, data transfer, and other related concerns are all handled by Modin under the hood."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 1: How to use Modin\n",
+    "\n",
+    "**GOAL**: Learn how to import Modin to accelerate and scale pandas workflows."
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
This should be under `Scale your pandas workflows by changing one line of code` section as for pandas on ray.

Reply:
@YarShev This line above